schematicbrushreborn-core/src/main/java/de/eldoria/schematicbrush/config/ConfigurationImpl.java

```diff
@@ -44,7 +50,22 @@ public void saveConfigs() {
 
     @Override
     public void reloadConfigs() {
-        presets = loadConfig(PRESET_FILE, null, false).getObject("presets", PresetRegistry.class, new PresetRegistryImpl());
+        Database database = new Database(Map.of("host",
```

Comment:
This is quite a mess in contrast to the implementation of the existing classes.

Also Object.requireNonNull is not really useful, since all it does is throwing a NPE, which would be thrown anyway. This is more an assertion than fix.